src/components/conversation/navigator.js

```diff
@@ -160,4 +162,35 @@ class Screen extends Component {
     )
   }
 }
+
+Screen.propTypes = {
+  editing: PropTypes.bool.isRequired,
+  selectedId: guid.isRequired,
+  personId: PropTypes.string.isRequired,
+  baseZoom: PropTypes.number.isRequired,
+  additionalLinks: PropTypes.array.isRequired,
+  w: PropTypes.number.isRequired,
+  h: PropTypes.number.isRequired,
+  x: PropTypes.number.isRequired,
+  y: PropTypes.number.isRequired,
+  zoomX: PropTypes.number.isRequired,
```

Comment:
I wonder if we actually need both `zoomX` and `zoomY`. It seems like if math works the way I think it does we could just get by with a single `zoomRatio`. We could handle that as an enhancement subsequent to this PR.